packages/schemas/src/text/pdfRender.ts

```diff
@@ -151,9 +150,11 @@ export const pdfRender = async (arg: PDFRenderProps<TextSchema>) => {
   }
 
   const pivotPoint = { x: x + width / 2, y: pageHeight - mm2pt(schema.position.y) - height / 2 };
+  const segmenter = new Intl.Segmenter(schema.locale, { granularity: 'grapheme' });
```

Comment:
Define this variable close to where it will be used.

This will be around bellow if block

> if (alignment === 'justify' && (lines[rowIndex + 1] ?? '' !== '')) {

Reply:
With that,`segmenter` will be created multiple times in the loop. Isn't that inefficient? Or no penallty on JS engines?

Reply:
Ah, got it. pls ignore this comment.